packages/cms-cli/commands/filemanager/fetch.js

```diff
@@ -0,0 +1,110 @@
+const { version } = require('../../package.json');
+
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { downloadFileOrFolder } = require('@hubspot/cms-lib/fileManager');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { resolveLocalPath } = require('../../lib/filesystem');
+
+const {
+  addConfigOptions,
+  addPortalOptions,
+  addLoggerOptions,
+  addOverwriteOptions,
+  setLogLevel,
+  getPortalId,
+} = require('../../lib/commonOpts');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { validatePortal } = require('../../lib/validation');
+const {
+  trackCommandUsage,
+  addHelpUsageTracking,
+} = require('../../lib/usageTracking');
+
+const FETCH_COMMAND_NAME = 'filemanager-fetch';
+const FETCH_DESCRIPTION =
+  'Download a folder or file from the HubSpot File Manager to your computer';
+
+const action = async ({ src, dest }, options) => {
+  setLogLevel(options);
+  logDebugInfo(options);
+
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
+
+  if (!validateConfig() || !(await validatePortal(options))) {
+    process.exit(1);
+  }
+
+  if (typeof src !== 'string') {
+    logger.error('A source to fetch is required');
+    process.exit(1);
+  }
+
+  dest = resolveLocalPath(dest);
+
+  const portalId = getPortalId(options);
+
+  trackCommandUsage(FETCH_COMMAND_NAME, null, portalId);
+
+  // Fetch and write file/folder.
+  downloadFileOrFolder(portalId, src, dest, options);
+};
+
+const command = 'fetch <src> [dest]';
+const describe = FETCH_DESCRIPTION;
+const handler = async argv => action({ src: argv.src, dest: argv.dest }, argv);
+const builder = yargs => {
+  addOverwriteOptions(yargs, true);
```

Comment:
The file manager doesn't have a concept of `draft` and `public` modes so this should be removed.

Reply:
Sure will do, it was already in there from the older commander version of the command so I just ported it over